### ISSUE TITLE
Redirect, revision and display title info disappears

### ISSUE THREAD

**Issue by e10km** [SemanticMediaWiki/SemanticBreadcrumbLinks#46]
### Setup and configuration

- SMW version: 2.5.0-3.0.0alpha
- SBL version: 1.4.0-1.5.0alpha
- MW version: 1.28.1-1.31.0
- DB (MySQL etc.): MySQL

### Issue

With SBL installed some information immediately below the page title disappears. Examples:

- Redirect information if the user entered the name of a page that is redirected to the current page.
- Revision information, including links to the next and the previous revision when viewing historical revisions of a page.
- DisplayTitle information when `$wgDisplayTitleHideSubtitle = false` when using the Display Title extension.

The problem is visible when entering pages via redirects or viewing old revisions of pages in  [sandbox](http://sandbox.semantic-mediawiki.org).

It seems to me that this function in SkinTemplateOutputModifier is the reason for the issue:
```
	public function modifyTemplate( &$template ) {
		// Always set subtitle to be empty when SBL is used to avoid any output
		// distraction
		$template->data['subtitle'] = '';
	}
```
When I commented out the line setting data['subtitle'] the missing information is displayed. What is the reason for emptying the subtitle for all cases?

**Comment by mwjames** [comment]
>   Redirect information if the user entered the name of a page that is redirected to the current page.
    Revision information, including links to the next and the previous revision when viewing historical revisions of a page.
    DisplayTitle information when $wgDisplayTitleHideSubtitle = false when using the Display Title extension.

@kghbln I'm unclear about those information, where should they appear? surely not on a wiki page or? Besides, the breadrcumbs aren't displayed on MW 1.30+ since something has changed in MW which needs fixing first!

**Comment by e10km** [comment]
### Example of missing redirect information
On wikipedia, if you go to the page [Munchen](https://en.wikipedia.org/w/index.php?title=Munchen) you will be redirected to the page [Munich](https://en.wikipedia.org/wiki/Munich). Immediately below the title you'll see the text:

From Wikipedia, the free encyclopedia
  (Redirected from [Munchen](https://en.wikipedia.org/w/index.php?title=Munchen&redirect=no))

...which indicates to the user that she has been redirected to this page.

Any redirect on the Sandbox page does not show the same redirect information.

**Comment by mwjames** [comment]
>Besides, the breadrcumbs aren't displayed on MW 1.30+ since something has changed in MW which needs fixing first!

In order to fix the display for MW 1.30+, #47 now uses `subtitle` to place the links instead of emptying it.

**Comment by kghbln** [comment]
Affirmative, works on sandbox:

![grafik](https://user-images.githubusercontent.com/1104078/45583746-b33e3000-b8c7-11e8-8496-f3416be3708f.png)
